lib/Utils.ts

```diff
@@ -154,3 +154,12 @@ export const groupBy = (arr: object[], keyGetter: (item: any) => string | number
 export const ms = (): number => {
   return Date.now();
 };
+
+/**
+ * Split listen string from config files.
+ */
+export const splitListen = (mergeTarget: any, listen: string) =>  {
```

Comment:
Passing a `mergeTarget` as argument to a function is inconsistent to what we are doing in the rest of the code base. Could you take care of replacing that logic with returning the actual values?